services/mirror/mirror_pull.go

```diff
@@ -411,6 +414,54 @@ func runSync(ctx context.Context, m *repo_model.Mirror) ([]*mirrorSyncResult, bo
 	return parseRemoteUpdateOutput(output, m.GetRemoteName()), true
 }
 
+// runSyncMisc runs the sync of Issues, Pull Requests, Reviews, Topics, Releases, Labels, and Milestones.
+// It returns true if the sync was successful.
+func runSyncMisc(ctx context.Context, m *repo_model.Mirror, lastSynced time.Time) bool {
+	repo := m.GetRepository()
+
+	remoteURL, remoteErr := git.GetRemoteURL(ctx, repo.RepoPath(), m.GetRemoteName())
+	if remoteErr != nil {
+		log.Error("SyncMirrors [repo: %-v]: GetRemoteAddress Error %v", m.Repo, remoteErr)
+		return false
+	}
+
+	password, ok := remoteURL.User.Password()
+	if !ok {
+		password = ""
+	}
+
+	opts := migration.MigrateOptions{
+		CloneAddr:      repo.OriginalURL,
+		AuthUsername:   remoteURL.User.Username(),
+		AuthPassword:   password,
+		UID:            int(repo.OwnerID),
+		RepoName:       repo.Name,
+		Mirror:         true,
+		LFS:            m.LFS,
+		LFSEndpoint:    m.LFSEndpoint,
+		GitServiceType: repo.OriginalServiceType,
+		Wiki:           repo.HasWiki() && m.SyncWiki,
+		Issues:         m.SyncIssues,
+		Milestones:     m.SyncMilestones,
+		Labels:         m.SyncLabels,
+		Releases:       m.SyncReleases,
+		Comments:       m.SyncComments,
+		PullRequests:   m.SyncPullRequests,
+		// Topics
+		// ReleaseAssets
+		MigrateToRepoID: repo.ID,
+		MirrorInterval:  m.Interval.String(),
+	}
+
+	_, err := migrations.SyncRepository(ctx, m.Repo.MustOwner(ctx), repo, opts, nil, lastSynced)
+	if err != nil {
+		log.Error("SyncMirrors [repo: %-v]: failed to sync repository: %v", m.Repo, err)
```

Comment:
SyncMirrors?

Reply:
Do you mean it should be the correct function name here? like `runSyncMisc`?